src/rust/bitbox02-rust/src/hww/api/ethereum/sighash.rs

```diff
@@ -90,18 +104,52 @@ fn hash_params<W: Write>(writer: &mut W, params: &Params) {
     }
 }
 
+fn hash_params_eip1559<W: Write>(writer: &mut W, params: &ParamsEIP1559) {
+    hash_u64(writer, params.chain_id);
+    hash_element(writer, params.nonce);
+    hash_element(writer, params.max_priority_fee_per_gas);
+    hash_element(writer, params.max_fee_per_gas);
+    hash_element(writer, params.gas_limit);
+    hash_element(writer, params.recipient);
+    hash_element(writer, params.value);
+    hash_element(writer, params.data);
+    hash_header(writer, RLP_SMALL_TAG, RLP_LARGE_TAG, 0_u16); // access list not currently supported and hashed as empty list
```

Comment:
`_u16` is not needed here, the `0` literal should work as `hash_header()` defines the type.